packages/components/dropdown/src/DropdownItems.tsx

```diff
@@ -9,6 +9,17 @@ interface ItemsProps {
   className?: string
 }
 
+/**
+ * HACKS
```

Comment:
I am not sure that we need to declare your solution as "hacky" :thinking: For me it is a simple bugfix, right?

Reply:
Yes, it's not exactly a hack, but I added this comment for two reasons. First, this bug cannot be reproduced locally, and secondly, we added this class to override styles computed not by us, but by an external library.

What would you suggest in place of HACK?

Reply:
I went with BUGFIX